### PR TITLE
[monitoring] Update overview dashboard

### PR DESCRIPTION
**Pull request by laushinka**
## Description
   - [x] Create sections in our Grafana overview e.g. general and external (workspace)
   - [x] Create a Grafana panel under the external dashboard for the workspace failure rate.  
   - [x] Edit websocket by clientType
   - [x] Add API Request error rate 
   - [x] Remove container image, cpu, memory, network (go into dashboard for pod health or services health),
   - [x] Add messagebus pod status, cpu, memory, network under a sub-section
   
## Related Issue(s)

Relates to #6785 

## How to test
Take a look at the [preview env](https://grafana-laushinka-grafana-6785.preview.gitpod-dev.com/d/Gj5DE-O7k/meta-overview).

## Release Notes

```release-note
NONE
```

## Documentation

/werft with-observability

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **laushinka** after the PR has been reviewed.

Associated issue: *#6785*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[operations/observability/mixins/meta/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/meta/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by laushinka** [comment]
@geropl Some of the panels won't show any data because the data isn't available in a preview env, e.g. "workspace failures" is filtered by `prod-ws.*`. Others need to be triggered to have data shown, e.g. "Websocket connections", "Team slot method calls", and "API request error rate". I ran all the queries against prod and they showed data, but please feel free to re-check them 🙏🏽

**Comment by geropl** [comment]
@laushinka For better reviewability I imported your dashboard into prod: https://grafana.gitpod.io/d/Gj5DE-O7k2/meta-overview-test?orgId=1&var-datasource=VictoriaMetrics&var-cluster=All Working on devstaging just does not make sense if the queries fail :roll_eyes: 

I also made some smaller changes:
 - applied `$cluster` to every query (I first struggled with, but think it's an excellent idea to leave it there!)
 - made variable `cluster` "reload on time range change" so we never run into the weird case where you have to reload the tab to see all data
 - switched all graphs from "Graph (old)" to new "Timeseries" (better rendered)
 - added "API Request Rate", and slightly re-ordered the tabs


What do you think? :open_mouth: 
[overview.json.txt](https://github.com/gitpod-io/gitpod/files/7593916/overview.json.txt)

**Comment by laushinka** [comment]
> @laushinka For better reviewability I imported your dashboard into prod: https://grafana.gitpod.io/d/Gj5DE-O7k2/meta-overview-test?orgId=1&var-datasource=VictoriaMetrics&var-cluster=All Working on devstaging just does not make sense if the queries fail 🙄
> 
> I also made some smaller changes:
> 
> * applied `$cluster` to every query (I first struggled with, but think it's an excellent idea to leave it there!)
> * made variable `cluster` "reload on time range change" so we never run into the weird case where you have to reload the tab to see all data
> * switched all graphs from "Graph (old)" to new "Timeseries" (better rendered)
> * added "API Request Rate", and slightly re-ordered the tabs
> 
> What do you think? 😮 [overview.json.txt](https://github.com/gitpod-io/gitpod/files/7593916/overview.json.txt)

@geropl Ahh, thanks! Next time I'll play around in prod! They look good 🤗 Also, how did you make the cluster reload on time range change? Learning lots here 👏🏼

**Comment by geropl** [comment]
> Also, how did you make the cluster reload on time range change?

@laushinka  It hides there: :point_down: 

image


Actually the variables config is quite hard to get right; especially the defaults are odd. For instance, if you allow "Include All option", which makes a lot of sense for us, it's important to specify a "Custom all value" of `.*`, so it matches all values. Also, all queries have to look like this: `cluster=~"$cluster"` (`=~` for patter matching) for this to work :roll_eyes: 
Re-learning this every time I work with Grafana :grimacing:

**Comment by laushinka** [comment]
> For instance, if you allow "Include All option", which makes a lot of sense for us, it's important to specify a "Custom all value" of `.*`, so it matches all values. Also, all queries have to look like this: `cluster=~"$cluster"` (`=~` for patter matching) for this to work 🙄 Re-learning this every time I work with Grafana 😬

@geropl Thanks for the notes! We'll keep learning I guess 🙈 And feel free to approve this PR!

**Comment by geropl** [comment]
> And feel free to approve this PR!

@laushinka The fixes (cmp. [here](https://github.com/gitpod-io/gitpod/pull/6834#issuecomment-977641299)) are contained in the dashboard in prod/linke in the comment above. Do you want to incorporate them into this PR? At least the query fixes and the variable config would be nice. Then I'll finally approve! :wink:

**Comment by laushinka** [comment]
> > And feel free to approve this PR!
> 
> @laushinka The fixes (cmp. [here](https://github.com/gitpod-io/gitpod/pull/6834#issuecomment-977641299)) are contained in the dashboard in prod/linke in the comment above. Do you want to incorporate them into this PR? At least the query fixes and the variable config would be nice. Then I'll finally approve! 😉

@geropl Totally forgot about that part lol. Will do so!

**Comment by geropl** [comment]
/hold

@laushinka Thx, especially with being so patient with me reviewing :grimacing: :pray: .

One last nit: If you adjust[ this `uid` back to `Gj5DE-O7k`, and turn the the `title` to something less "wip"/"test"]( https://github.com/gitpod-io/gitpod/pull/6834/files#diff-20bc0ecaa1e5da3a00db9476b8a3b9f28375569e82c0e153f516d4033dc6b2acR1051-R1052) we don't have to adjust that later.
Approving anyway as that's minor! :+1:

**Review by geropl — APPROVED** [review]
:rocket:

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 6489f235c45db7d37cb3fa5adb99462028ef56a4</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6834#pullrequestreview-815723171" title="Approved">geropl</a>*

Associated issue: *#6785*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[operations/observability/mixins/meta/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/meta/OWNERS)~~ [geropl]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by laushinka** [comment]
> One last nit: If you adjust[ this `uid` back to `Gj5DE-O7k`, and turn the the `title` to something less "wip"/"test"]

@geropl Ohh, that's a good tip!

**Comment by roboquat** [comment]
New changes are detected. LGTM label has been removed.

**Comment by laushinka** [comment]
@geropl I pushed so LGTM is removed 🙈

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 0815892e0c73ff31d707ddf9b8306d0a588ab42d</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6834#issuecomment-979869795" title="LGTM">geropl</a>*

Associated issue: *#6785*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[operations/observability/mixins/meta/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/meta/OWNERS)~~ [geropl]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
